cfme/tests/control/test_actions.py

```diff
@@ -426,7 +426,7 @@ def test_action_prevent_event(request, vm, vm_off, policy_for_testing):
         pytest.fail("CFME did not prevent starting of the VM {}".format(vm.name))
 
 
-@pytest.mark.meta(blockers=[1439331])
+@pytest.mark.meta(blockers=[BZ(1439331)])
```

Comment:
Why do you keep his BZ? It's `CLOSED CURRENTRELEASE`

Reply:
I just fixed bug. I didn't even check this. I think this is test owner responsibility.